README.md

```diff
@@ -2,9 +2,19 @@
 
 This repo deals with the high-level configuration of Cloud Services. `main.yml` contains the source of truth for CS apps, and the `akamai` folder deals with updating our Akamai configuration.
 
+## Branch syncing
+
+These branches sync:
+
+* ci-beta -> nightly-beta -> qa-beta (and qaprodauth-beta) -> stage-beta
+* ci-stable -> nightly-stable
+* qa-stable (and qaprodauth-stable) -> stage-stable
```

Comment:
qaprodauth-stable isn't a branch, FYI. If there's misinformation about the qaprodauth env, this section may not be the right place to elaborate